### DISCUSSION TITLE
Comments are missed when convert issue to discussion

### DISCUSSION THREAD

**Discussion by dingmeng-xue** [community/community#3022] [Discussions]
I found all comment history are removed when I convert one issue to discussion. 

Is it possible that I can find them? 

Is it a feature request that comments should be converted together with the issue?

**Comment by becca** [comment]
We're working on fixing this bug right now.

**Reply by becca** [reply]
Do you have a list of the issues/discussions that were converted and have lost comments? If I could have that list of issues we can make sure to recover the lost comments @dingmeng-xue

**Reply by dingmeng-xue** [reply]
I only tried this one. Original version cannot be found now.

https://github.com/Azure/azure-powershell/discussions/13703

**Reply by becca** [reply]
When you say `original version` are you saying that original comments are missing/were deleted or something different?

**Reply by dingmeng-xue** [reply]
#13703 was Git issue and couple users had put comment. When I converted into Github Discussion, the comment cannot be seen. Current comments on #13703 were added after it was converted to discussion.

**Reply by becca** [reply]
We fixed this issue and restored all lost comments from the issue/discussion listed above. 🙇
